dateparser_cli/utils.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import os
+from pathlib import Path
+import logging
+
+DEFAULT_UNIX_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", '~/.cache')
+DEFAULT_WIXDOWS_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", os.path.join(str(Path.home()), "AppData", "Roaming"))
+DEFAULT_DIR_NAME = os.environ.get("DEFAULT_DIR_NAME", 'date_parser_models')
+
+date_parser_model_home = None
```

Comment:
```suggestion
dateparser_model_home = None
```

(and references below and in function names)

Reply:
Thanks, I have made the changes 👍🏻